frappe/core/doctype/communication/communication.py

```diff
@@ -387,13 +387,13 @@ def parse_email(communication, email_strings):
 		if email_string:
 			for email in email_string.split(","):
 				if delimiter in email:
-					email = email.split("@")[0]
-
-					doctype = unquote(email.split(delimiter)[1])
-					docname = unquote(email.split(delimiter)[2])
-
-					if doctype and docname and frappe.db.exists(doctype, docname):
-						communication.add_link(doctype, docname)
+					email_local_part = email.split("@")[0]
```

Comment:
```suggestion
					email_username = email.split("@")[0]
```

Reply:
@hrwX The actual term for the part of an email address before the '@' symbol is Local-part [see Wikipedia entry: https://en.wikipedia.org/wiki/Email_address#Local-part](https://en.wikipedia.org/wiki/Email_address#Local-part)

Especially since the purpose of this particular function is to try and split apart the string and deduce the username (if there is one), it would be especially confusing to anyone working on this in the future.

FWIW: I did initially have a variable for 'email_username' but since it is not used my PR got kicked back

Reply:
@nobesnickr Sounds good.